api/src/main/java/org/bahmni/module/teleconsultation/api/TeleconsultationService.java

```diff
@@ -0,0 +1,9 @@
+package org.bahmni.module.teleconsultation.api;
+
+import org.openmrs.api.OpenmrsService;
+
+
+public interface TeleconsultationService extends OpenmrsService {
+	
+	public String generateTeleconsultationLink(String uuid);
```

Comment:
can you also add appropriate `Privilege` to access this service ?

Reply:
Updated